docs/releases/v3.2.1.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: -4
+---
+
+# CSS v3.2.1
+
+### New Features 🎉
+
+-   Added Client Feedback section in [Common Hosted Single Sign-on (CSS)](https://bcgov.github.io/sso-requests) home page
+-   Updated wiki link with usage tracking
+-   Updated to ensure Keycloak login title to be case sensitive, aligned with field "Pathfinder SSO Login Page Name". This change gives clients flexibility, before this change, Keycloak login page title is in upper case only.
+
+
+### Bug fixes and enhancement 🐛
+
+-   Temporarily disable Additional Role Attribute (optional) setting for new SAML integrations. We discovered this feature behaves inconsistently among environments. We will work with RedHat to find the root cause. Exiting integrations with this optional set up will not be affected.
```

Comment:
A typo `Existing`

Reply:
Good catch! I ran this through Word, but since it's a valid word, it didn't report :)

Reply:
Typo fixed @NithinKuruba